### DISCUSSION TITLE
meeting guide shows not supported

### DISCUSSION THREAD

**Discussion by Gary4593** [code4recovery/12-step-meeting-list#580] [Q&A] [answered]
My wordpress is the latest version and meeting guide shows not supported and if I add a meeting and publish its status is DRAFT

**Comment by brianw-area24** [comment, marked as answer]
Although the plugin page on WordPress.org doesn't show it, the 12 Step Meeting List (TSML) plugin does work with with the latest version of WordPress.

In my experience, when a new meeting gets switched to Draft mode, it's because one of the or more of the fields aren't filled out completely, usually something with the address.

If you can post here what information you're trying to put in, I would be happy to do more testing for you.

Brian W
